### PR TITLE
New branch for dir built-in function

### PR DESCRIPTION
**Pull request by VasilisCbt17gr**
### Description

- recreate dir() function
- create new branch
- closes issue #2470 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by CBID2 — APPROVED** [review]
Looks good to me

File: content/python/concepts/built-in-functions/terms/dir/dir.md
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/dir/dir.md:25]: Remove this header and the following blank line so the sentence below is in the syntax section.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@VasilisCbt17gr I had one comment.

File: content/python/concepts/built-in-functions/terms/dir/dir.md
Comment (VasilisCbt17gr) [reply content/python/concepts/built-in-functions/terms/dir/dir.md]: @SSwiniarski  I modified it. Could you check it?

**Review by SSwiniarski — APPROVED** [review]
@VasilisCbt17gr Looks good for a second review.

**Comment by VasilisCbt17gr** [comment]
@SSwiniarski And what are my next steps? May i have to click 'Update branch'? And another question, All checks are in status " Expected". What Shoulod i do to complete them?

**Comment by SSwiniarski** [comment]
@VasilisCbt17gr you just have to wit for another maintainer to come and do a second review of your entry. As for the checks, a maintainer has to approve them to run on a first time PR. I'll do that now.

File: content/python/concepts/built-in-functions/terms/dir/dir.md
Comment (CBID2) [inline content/python/concepts/built-in-functions/terms/dir/dir.md:37]: [Typo]
```suggestion
The following output will look like below:
```

File: content/python/concepts/built-in-functions/terms/dir/dir.md
Comment (CBID2) [inline content/python/concepts/built-in-functions/terms/dir/dir.md:45]:
```suggestion
Use `dir()` to return both the methods made available by default to all classes defined in Python and the user-defined properties of the class `Phone`:
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @VasilisCbt17gr! :) I just found two typos in your entry. Other than that, you're off to a great start. Keep it up! :)

**Comment by VasilisCbt17gr** [comment]
@CBID2  Could you help me with an issue? i corrected the typos and i commited the changes. However my commit does not appear in commits on github site.

**Comment by CBID2** [comment]
> @CBID2  Could you help me with an issue? i corrected the typos and i commited the changes. However my commit does not appear in commits on github site.

Sure @VasilisCbt17gr. Post the link to it in your reply. :)

**Comment by VasilisCbt17gr** [comment]
@CBID2  i executed git commit in vs code, not online. What i wanted to ask you is what do you think i'm doing wrong and my commit does not appear online?

**Comment by CBID2** [comment]
> @CBID2  i executed git commit in vs code, not online. What i wanted to ask you is what do you think i'm doing wrong and my commit does not appear online?

Did you make the commit on  the branch you used to do the PR?

**Comment by VasilisCbt17gr** [comment]
@CBID2 yes, i switch to the created branch, i corrected the typos, adding the modifed file and finally i commited the changes.
image

**Comment by CBID2** [comment]
> @CBID2 yes, i switch to the created branch, i corrected the typos, adding the modifed file and finally i commited the changes.
> 
> image
> 
> 

Were you logged into GitHub when you made those commits @VasilisCbt17gr?

**Comment by VasilisCbt17gr** [comment]
@CBID2  Yes, i'm still logged in.

**Comment by CBID2** [comment]
> @CBID2  Yes, i'm still logged in.

Ok, did your sync your local repo with the remote version @VasilisCbt17gr, meaning the repo that's on GitHub?

**Comment by VasilisCbt17gr** [comment]
@CBID2 No. Have i to run " git fetch"  ?

**Comment by CBID2** [comment]
> @CBID2 No. Have i to run " git fetch" ?

I don't think so @VasilisCbt17gr. Check out [GitHub's tutorial](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/keeping-your-local-repository-in-sync-with-github) on this. In the future, I recommend downloading these extensions:
1. GitHub Pull Requests: https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github
2. GitHub Desktop: https://desktop.github.com/

**Comment by VasilisCbt17gr** [comment]
@CBID2  Thank you for your support!! I'm gonna follow your guidance.

**Comment by CBID2** [comment]
> @CBID2  Thank you for your support!! I'm gonna follow your guidance.

You're welcome @VasilisCbt17gr! :)

**Comment by VasilisCbt17gr** [comment]
@CBID2  Could you check the changes?

**Review by caupolicandiaz — APPROVED** [review]
@VasilisCbt17gr @SSwiniarski LGTM

**Pull request merged**
